src/redux/missions/missionSlice.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import fetchMission from './missions';
+
```

Comment:
While you did a good job working on the project requirements I notices you disabled eslint. Its against microverse policy to disable linters because linters help us write clean code. It would be nice if you kindly enable eslint.